src/d/d_attention.cpp

```diff
@@ -453,19 +655,72 @@ BOOL sound_attention(fopAc_ac_c* actor, void* userWork) {
 void dAttention_c::runSoundProc() {
     mEnemyBsPcId = fpcM_ERROR_PROCESS_ID_e;
     mEnemyDistance = 10000.0f;
-    if (!chkFlag(0x80000000)) {
+    if (!chkFlag(AttnFlag_80000000)) {
         fopAcIt_Executor((fopAcIt_ExecutorFunc)sound_attention, this);
         fopAc_ac_c* actor = fopAcM_SearchByID(mEnemyBsPcId);
         if (actor != NULL) {
             mDoAud_bgmNowBattle(mEnemyDistance * 0.1f);
-            setFlag(0x100);
+            setFlag(AttnFlag_00000100);
         }
     }
 }
 
+// This is some combination of flags `g_dComIfG_gameInfo.play.mPlayerStatus[0][0]`
+// can take, seemingly related to "judgement".
+#define PLAYER_STATUS_FLAG_MAGIC_JUDGEMENT 0x37a02371
+
 /* 8009EB38-8009EDB8       .text runDrawProc__12dAttention_cFv */
 void dAttention_c::runDrawProc() {
-    /* Nonmatching */
+    /* TODO: Magic constants */
+    if (this->chkFlag(AttnFlag_00000008)) {
```

Comment:
```suggestion
    if (chkFlag(AttnFlag_00000008)) {
```
Typing `this->` is unnecessary. This applies everywhere in this file